herbert_core/utilities/maths/vals2distr.m

```diff
@@ -31,9 +31,9 @@
 flags = {'norm','poisson'};
 [par,keyval,~,~,ok,mess] = parse_arguments (varargin,keyval_def,flags);
 if ~ok, error(mess), end
-if numel(par)>1
-    error('Check number of input arguments')
-end
+% if numel(par)>1
```

Comment:
If this is erroneous why not delete it?

Reply:
THis is really me just checking that I can check in, run Jenkins, so I am just commenting out and will edit properly later (there are other changes to be performed)

I.e. testing Toby understands what is going on!

Reply:
...and it seems that I am not a member of the project on Jenkins. I can log into Jenkins, but do not see results of this test.